bear-requirements.yaml

```diff
@@ -231,3 +231,4 @@ r_script_requirements:
     version: '>1.5'
   lintr:
     version: '>=1.0.2'
+    
```

Comment:
why is this in your diff ????

Reply:
@sladyn98 This is not a newcomer task, so you should be able to use `git ` properly, and people would not give you hints on `git` anymore. I remember in this PR, you add irrelevant stuff multiple times, and every time you say you don't add them 😅 Before you commit or push, verify your local diff is correct.

Reply:
This is still here.

before you push, run `git show` locally.  Read it, top to bottom.  Everything in there is your change.  If it doesnt belong there, do not push.  Fix it.